### PR TITLE
Sru/20.2.45 softlayer verification results and script refactor

### PR DESCRIPTION
**Pull request by blackboxsw**
Softlayer only has Xenial and Bionic. Softlayer currently only has Ubuntu Precise, Trusty, Xenial and Bionic . No Focal images yet so this test will be sparse. basically a sanity check to make sure we haven't degraded across upgrade path.

### CODE REVIEW

File: sru-scripts/manual/softlayer-sru [hunk 1]
```diff
@@ -32,11 +32,12 @@ create_setup_proposed_script
  
 echo "### BEGIN $UBUNTU_RELEASE"
 VM_IP=`launch-softlayer --image=os:$UBUNTU_RELEASE -u $SAMPLE_CLOUDCONFIG | awk '/primary ip/{printf "root@%s", $NF}'`
-sleep 180
+sleep 100
```
Comment (blackboxsw) [inline sru-scripts/manual/softlayer-sru:35]: Reduced this count as we now run retry_ssh_command() which handles remaining sleeps

File: manual/softlayer-sru-20.2.45.txt [hunk 2]
```diff
@@ -0,0 +1,1161 @@
...
+  "errors": []
+ }
+}
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- '!' grep Trace /var/log/cloud-init.log
++ retry_ssh_command root@75.126.139.189 systemd-analyze 5
++ local vm_ip=root@75.126.139.189 cmd=systemd-analyze sleep_seconds=5
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- systemd-analyze
+Startup finished in 6.608s (kernel) + 3min 6.909s (userspace) = 3min 13.517s
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.139.189 -- systemd-analyze
+Startup finished in 6.608s (kernel) + 3min 6.909s (userspace) = 3min 13.517s
```
Comment (lucasmoura) [inline manual/softlayer-sru-20.2.45.txt:105]: Is this expected for xenial, a boot time of 3 min ? I know that this is happening on both current and proposed versions. but I still think it is odd

File: manual/softlayer-sru-20.2.45.txt [hunk 3]
```diff
@@ -0,0 +1,1161 @@
++ SRU_VARS=./sru-scripts/sru-vars.template
```
Comment (lucasmoura) [inline manual/softlayer-sru-20.2.45.txt:1]: Is there a reason to not attach the scrip that generated the output here ?

File: manual/softlayer-sru-20.2.45.txt (hunk 2 above)
Comment (blackboxsw) [reply manual/softlayer-sru-20.2.45.txt:105]: Yeah crazy right. softlayer on xenial I believe has always taken 3 mins to boot. I'm not certain what their backplane provisioning issue is here, but I don't think we've dug into it on this platform.

File: manual/softlayer-sru-20.2.45.txt (hunk 3 above)
Comment (blackboxsw) [reply manual/softlayer-sru-20.2.45.txt:1]: +1 added the appropriate script header

**Pull request merged**